docs/advanced-techniques.md

```diff
@@ -227,6 +227,120 @@ A merged module's code is included in the resulting application's bundles.
 
```

Comment:
I can't seem to edit the lines above, so can you please make the following changes?

Line 29: `They abstract away all direct interactions with the underlying storage, allowing reSolve to provide a unified data management API.`

Please rewrite without using "provide".

Line 36: `Resolve comes with a set of adapters covering popular DBMS choices.`

I'd like to avoid the word "covering", and make it more specific. For example, do you mean:

`Resolve includes a set of adapters compatible with popular DBMS choices.`

Reply:
>Please rewrite without using "provide".
Rewrote the sentence like this: "This allows reSolve to abstract away all direct interactions with the underlying storage and expose a unified data management API."
>I'd like to avoid the word "covering", and make it more specific. For example, do you mean:
I applied your version, thank you.

Reply:
> Rewrote the sentence like this: "This allows reSolve to abstract away all direct interactions with the underlying storage and expose a unified data management API."

Thank you. Sounds good. Please apply your suggestion.

Reply:
Oh, I see that you've already applied it. Thank you.